### DISCUSSION TITLE
Loss function choice for extrem events

### DISCUSSION THREAD

**Discussion by rhedouin** [neuralhydrology/neuralhydrology#200] [General]
Hello, 

I'm trying to train a model to predict extreme event. There is several evaluation metrics well adapted to this purpose (such as missed peaks or mean absolute percentage peak_error). However, there is no similar metric that could be used as a loss function to train the model. I assume this is because the loss function is calculated consistently and therefore cannot be too complex, like those metrics that have to find the peak first ... that would take too long. So my question is this: in your opinion, what loss is best suited to predicting extreme flows (and floods in the end)? I have implemented a Peak error swish (PES) metric that I found in this article ‘Enhanced LSTM Model for Daily Runoff Prediction in the Upper Huai River Basin, China’, it is slightly better than a MSE but not much.

Thanks in advance for any help you can give me.
